src/Shared/HttpSys/RequestProcessing/HeaderEncoding.cs

```diff
@@ -27,4 +28,21 @@ internal static byte[] GetBytes(string myString)
     {
```

Comment:
Is this `GetBytes` method still used anywhere after these changes?

Reply:
Yes it is used in a few places. I am working on removing it but wanted a review of the current approach first.